File: udemy/java/TabuadaDoWhile/src/tabuadaDoWhile/TabuadaDoWhile.java
package tabuadaDoWhile;
import javax.swing.JOptionPane;
public class TabuadaDoWhile {
	public static void main(String[] args) {
		Integer num, i = 1;
		String mensagem ="";
		num = Integer.parseInt(JOptionPane.showInputDialog("informe um valor para imprimir a tabuada: "));
		do {
			mensagem = mensagem + num+"x"+i+" = "+num*i+"\n";
			i++;
		}
		while(i<=10);
		JOptionPane.showMessageDialog(null, mensagem);
	}
}

File: udemy/pascal/projetos/TabuadaDoWhile.pas
program TabuadaDoWhile;
var
    num, i: Integer;
begin
  writeln ('informe um numoero para imprimir a tabuada: ');
  read(num);
  i := 1;
  repeat
  begin
    writeln(num, 'x',i, ' = ', num*i);
    i := i+1;
    end;
    until(i>10);
end.
